tests/cypress/e2e/unit_tests/rbac_fleet.spec.ts

```diff
@@ -268,5 +268,257 @@ describe('Test Fleet access with RBAC with custom roles using Standard User', {
+      
+      const stduser = "std-user-48"
+      
+      // Create "Standard User"
+      cypressLib.burgerMenuToggle();
+      cypressLib.createUser(stduser, uiPassword);
+
+      // Assign role to the created user
+      cy.assignRoleToUser(stduser, customRoleName_3)
+
+      // Logout as admin and login as other user
+      cypressLib.logout();
+      cy.login(stduser, uiPassword);
+
+      // CAN go to Continuous Delivery Dashboard and "list" gitrepos
+      cy.accesMenuSelection('Continuous Delivery', 'Dashboard');
+      cy.get("div[data-testid='collapsible-card-fleet-local']").contains(repoName).should('be.visible');
+      cy.get("div[data-testid='collapsible-card-fleet-default']").contains(repoNameDefault).should('be.visible');
+      
+      // CHECKS IN  FLEET-DEFAULT
+      // CAN "Create" and "Edit"
+      cy.accesMenuSelection('Continuous Delivery', 'Git Repos');
+      cy.clickButton('Add Repository');
+      cy.contains('Git Repo:').should('be.visible');
+      cy.clickButton('Cancel');
+      cy.open3dotsMenu(repoNameDefault, 'Edit Config');
+      cy.clickButton('Cancel');
+      // Can`t "Delete"
+      cy.open3dotsMenu(repoNameDefault, 'Delete', true);
+
+      // CHECKS IN  FLEET-LOCAL
+      cy.fleetNamespaceToggle('fleet-local');
+      // CAN "Edit"
+      cy.open3dotsMenu(repoName, 'Edit Config');
+      cy.clickButton('Cancel');
+      // Can`t "Delete"
+      cy.open3dotsMenu(repoName, 'Delete', true);
+    })
+  )
+
+  qase(50,
+    it('Fleet-50: Test "Standard-user" | Custom Role | Fleetworkspaces, Bundles = [ALL] | GitRepos = [List, Delete]', { tags: '@fleet-50' }, () => {
+      
+      const stduser = "std-user-50"
+      
+      // Create "Standard User"
+      cypressLib.burgerMenuToggle();
+      cypressLib.createUser(stduser, uiPassword);
+
+      // Assign role to the created user
+      cy.assignRoleToUser(stduser, customRoleName_4)
+
+      // Logout as admin and login as other user
+      cypressLib.logout();
+      cy.login(stduser, uiPassword);
+
+      // CAN go to Continuous Delivery Dashboard and "list" gitrepos
+      cy.accesMenuSelection('Continuous Delivery', 'Dashboard');
+      cy.get("div[data-testid='collapsible-card-fleet-local']").contains(repoName).should('be.visible');
+      cy.get("div[data-testid='collapsible-card-fleet-default']").contains(repoNameDefault).should('be.visible');      
+      // CHECKS IN  FLEET-DEFAULT
+      cy.accesMenuSelection('Continuous Delivery', 'Git Repos');       
+      // Can't "Create" repos    
+      cy.get('.btn.role-primary').contains('Add Repository').should('not.exist');
+      // Cant't "Edit"
+      cy.open3dotsMenu(repoNameDefault, 'Edit Config', true);
+      // CAN "Delete"
+      cy.open3dotsMenu(repoNameDefault, 'Delete');
+      cy.clickButton('Cancel');
+
+      // CHECKS IN  FLEET-LOCAL
```

Comment:
```suggestion
      // CHECKS IN FLEET-LOCAL
```

Reply:
Thanks, addressed in `3f474bf958123b38439a2f4d1683ac3dcf0b3f16`